From: Dan Streetman <ddstreet@canonical.com>
Origin: vendor
Forwarded: not-needed
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/corosync/+bug/1918735

Set allow_knet_handle_fallback to yes in example conf file. This setting is
required for transport=knet (which is the default) to work in an unprivileged
container.

--- a/conf/corosync.conf.example
+++ b/conf/corosync.conf.example
@@ -1,4 +1,10 @@
 # Please read the corosync.conf.5 manual page
+system {
+	# This is required to use transport=knet in an unprivileged
+	# environment, such as a container. See man page for details.
+	allow_knet_handle_fallback: yes
+}
+
 totem {
 	version: 2
 
